content/docs/tutorials/get-started/experiments.md

```diff
@@ -1,173 +1,217 @@
+numbers that are supposed to be visualized to compare or make sense of them.
 
-Let's now redefine the featurization stage so that DVC knows that it depends on
-the specific values of `max_features` and `ngram_range`. For this we use the
-`-p` (`--params`) option of `dvc run`. `params.yaml` is the default parameters
-file name in DVC, so there's no need to specify this:
+`cache: false` means that those file are small enough and versioned directly
+with Git.
 
-```dvc
-$ dvc run -y -f featurize.dvc \
-          -d src/featurization.py -d data/prepared \
-          -p max_features,ngram_range.lo,ngram_range.hi \
-          -o data/features \
-          python src/featurization.py \
-                 data/prepared data/features params.yaml
+</details>
 
-$ git add featurize.dvc
-$ git commit -m "Update featurization stage"
+DVC expects `evaluate.py` to
+[write](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L35)
+model
+[AUC](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
+value to the `scores.json` file which is marked as a metrics file with `-M`:
+
+```json
+{ "auc": 0.57313829 }
 ```
 
-> Please refer to `dvc params` for more information.
+And it
+[writes](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L31)
+`precision`, `recall`, `thresholds` arrays it gets with the
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
+call into the `prc.json`:
+
+```json
+{
+  "prc": [
+    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
+    ...{ "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
+  ]
+}
+```
 
-### Run the experiment
+> DVC doesn't force you to use any specific file names, or even format or
+> structure of a metrics or plots file - it's pretty much user and case defined,
+> please, refer to `dvc metrics` and `dvc plots` documentation for more details.
 
-Let's [reproduce](/doc/tutorials/get-started/data-pipelines#reproduce) our
-pipeline up to the model training now:
+Let's save this experiment to compare it later using DVC comparison commands:
 
 ```dvc
-$ dvc repro train.dvc
-$ git commit -am "Reproduce model using bigrams"
+$ git add scores.json prc.json
+$ git commit -a -m "Create evaluation stage"
 ```
 
-> Notice that `git commit -a` stages all the changes produced by `dvc repro`
-> before committing them with Git. Refer to the
-> [command reference](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--a)
-> for more details.
+We've managed to capture metrics that experiment has, and later we will see how
+this information can be used to compare different experiment iterations,
+visualize them (e.g. `ROC` or model loss plots). Let's see how can we capture
+another important piece of information that will be important to compare
+experiments - experiment parameters.
```

Comment:
metrics from an experiment. Later we...

compare different experiment iterations (e.g. `ROC` or model loss plots) and visualize them.

Reply:
"Let's see how can we capture" -> move this one to after the next H2?